ui/plugins/controller.js

```diff
@@ -76,11 +77,102 @@ treeherder.controller('PluginCtrl', [
+                            // relevant all_others bugs, so don't show them either.
+                            !suggestion.bugs.too_many_open_recent
+                    );
+                });
+
+                // if we have no bug suggestions, populate with the raw errors from
+                // the log (we can do this asynchronously, it should normally be
+                // fast)
+                if (!suggestions.length) {
+                    ThTextLogStepModel.query({
+                        project: $rootScope.repoName,
+                        jobId: $scope.job.id
+                    }, function (textLogSteps) {
+                        $scope.errors = textLogSteps
+                            .filter(step => step.result !== 'success')
+                            .map(function (step) {
+                                return {
+                                    name: step.name,
+                                    result: step.result,
+                                    lvURL: getLogViewerUrl($scope.job.id, $rootScope.repoName, step.finished_line_number)
+                                };
+                            });
+                    });
+                }
+                $scope.suggestions = suggestions;
+                $scope.bugSuggestionsLoading = false;
+            });
+        };
+
+        $scope.fileBug = function (index) {
+          const summary = $scope.suggestions[index].search;
+          const crashRegex = /application crashed \[@ (.+)\]$/g;
+          const crash = summary.match(crashRegex);
+          const crashSignatures = crash ? [crash[0].split("application crashed ")[1]] : [];
+          const allFailures = $scope.suggestions.map(sugg => (sugg.search.split(" | ")));
+
+          const modalInstance = $uibModal.open({
+            template: intermittentTemplate,
+            controller: 'BugFilerCtrl',
+            size: 'lg',
+            openedClass: "filer-open",
+            resolve: {
+              summary: () => (summary),
+              search_terms: () => ($scope.suggestions[index].search_terms),
+              fullLog: () => ($scope.job_log_urls[0].url),
+              parsedLog: () => ($scope.lvFullUrl),
+              reftest: () => ($scope.isReftest() ? $scope.reftestUrl : ""),
+              selectedJob: () => ($scope.selectedJob),
+              allFailures: () => (allFailures),
+              crashSignatures: () => (crashSignatures),
+              successCallback: () => (data) => {
+                // Auto-classify this failure now that the bug has been filed
+                // and we have a bug number
+                thPinboard.addBug({ id: data.success });
+                $rootScope.$evalAsync(
+                  $rootScope.$emit(
+                    thEvents.saveClassification));
+                // Open the newly filed bug in a new tab or window for further editing
+                window.open("https://bugzilla.mozilla.org/show_bug.cgi?id=" + data.success);
```

Comment:
I know this is pre-existing, but perhaps we should still switch it to using the helper whilst we're here?
https://github.com/mozilla/treeherder/blob/31dc8dec9918b19fccad8054fc47a784dec8cfcc/ui/helpers/urlHelper.js#L123-L126

Reply:
Ahh yes, of course.  :)